.gitignore

```diff
@@ -13,4 +13,10 @@
 .env.local
 
 # Terraform artifacts
-.terraform
+**/.terraform/**
+examples/*/plan.save
```

Comment:
@aliscott now that we've removed the need for creds in examples, should we just delete this and the plan.json file from the repo? the README could just show `--tfdir`.
Having these files in the repo is slightly annoying due to the conflicts they create and I always check my ones don't have any AWS creds before I push...

Reply:
Makes sense